### PR TITLE
Allow requesting NOMIS slots from PVB api

### PR DESCRIPTION
**Pull request by mloughran**
Hope you enjoy the super secret method of toggling this feature :)

### CODE REVIEW

File: app/models/slots_step.rb [hunk 1]
```diff
@@ -41,6 +41,9 @@ def options
   end
 
   def slot_constraints
-    @constraints ||= processor.booking_constraints.on_slots
+    # Temporary Easter egg to switch on live slot availability :)
+    show_live_slots = processor.prisoner_step.first_name == 'Rickie'
```
Comment (tatyree) [inline app/models/slots_step.rb]: "Hey Rickie you're so fine"???

File: spec/services/prison_visits/api_spec.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
   # the VCR cassettes. Sadly this will vary from system to system as a result
   # of using UUIDs for prisons.
   let(:cardiff_prison_id) { '0614760e-a773-49c0-a29c-35e743e72555' }
+  let(:leeds_prison_id) { '8076d43d-429d-4e19-aac1-178ff9d112d3' }
```
Comment (tatyree) [inline spec/services/prison_visits/api_spec.rb]: It shouldn't: 

https://github.com/ministryofjustice/prison-visits-2/blob/master/db/seeds/prison_uuid_mappings.yml

**Comment by tatyree** [comment]
LGTM.

File: spec/services/prison_visits/api_spec.rb (hunk 2 above)
Comment (mloughran) [inline spec/services/prison_visits/api_spec.rb]: @tatyree I stand corrected! I still don't like the UUIDs though ;)

**Pull request merged**